keras_cv/models/object_detection/yolo_v8/yolo_v8_backbone_presets.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""YOLOv8 Backbone presets."""
+
+backbone_presets_no_weights = {
+    "yolov8_n_backbone": {
+        "metadata": {
+            "description": ("An extra small YOLOV8 backbone"),
+        },
+        "class_name": "keras_cv.models>YOLOV8Backbone",
+        "config": {
```

Comment:
Include `activation` in all the presets. We don't want all the presets invalidated if a default arg changes.